lib/same/interactor/context_attributes.rb

```diff
@@ -46,6 +45,16 @@ def define_field_methods(fields, default:)
         before :assert_context_attributes
       end
 
+      def _fetch_context_method_value(field, default)
```

Comment:
Usually underscore prefix means that method/variable is unused. 
Please, use 
```suggestion
      def fetch_context_method_value(field, default)
```